bigtop-packages/src/common/hadoop/install_hadoop.sh

```diff
@@ -177,8 +177,28 @@ export HADOOP_LIBEXEC_DIR=/${SYSTEM_LIBEXEC_DIR#${PREFIX}}
 exec ${component#${PREFIX}} "\$@"
 EOF
   chmod 755 $wrapper
+  
+  # ODPI-131 Add wrappers for hdfs, yarn, mapred in $HADOOP_DIR
```

Comment:
@alanfgates @rvs Changed this to a symbolic link instead. Would that be a better approach?